Scripts/Swift/SetUpDeliveryConstants.swift

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/swift
+
+import Foundation
+
+let fileManager = FileManager.default
+
+print("Do you want to set up Constants values? (Can be edited later) [Y/n]: ")
+
+var arg = "y"
+
+arg = readLine() ?? arg
+
+switch arg.lowercased() {
+    case "y", "yes":
+        try safeShell("open -a Xcode fastlane/Constants/Constant.swift")
```

Comment:
I don't try this yet, but the script might exit with a non-human-readable message (e.g., "EXIT with xxx code").

IMO, we should catch the error and exit gracefully with a helpful message like "Could not open Xcode. Please try again" 🙏

Reply:
Good suggestion. Updated this part, this is the demo of the script failing. <img width="571" alt="Screen Shot 2023-07-17 at 17 46 33" src="https://github.com/nimblehq/ios-templates/assets/6356137/70cf6c36-4603-4977-8574-6796995537f9">